pages/covid-19/sarscov2_submission.md

```diff
@@ -146,18 +145,30 @@ For submission of a large number of files, we recommend to use the Excel templat
 
 Use the *_alias* field on each sheet to interlink the experiments, studies, runs, samples and files with each other. The template can be downloaded, completed and uploaded using the Galaxy upload tool.
 
-Complete the metadata template in your computer. **All fields of the template must be complete**. [Here](https://drive.google.com/file/d/1Z3LszV6IkgmcESsz2K7Mdv8pcnolUHT5/view?usp=sharing) you can find an example of part of the metadata associated with an actual submission to ENA (accession number PRJEB40711). Upload the metadata file to Galaxy, and select it on **'File based on templates here:'**.
-Finally, select the human-filtered data files associated with the metadata, fill in the Affiliation center and click on 'Execute'.
+
+#### Spreadsheet template upload
+
+ For submission of a large number of files, we recommend to use the Excel template to upload the metadata. The spreadsheet contains one worksheet each for study, sample, experiment and run metadata. A [template repository](https://github.com/ELIXIR-Belgium/ENA-metadata-templates) is in place that supports a spreadsheet with controlled vocabulary build in for every checklist on ENA. 
 
 
 #### Interactive metadata upload
 For a small number of submissions, metadata is best entered interactively in Galaxy using the submission tool. Metadata fields are nested according to ENA metadata model described above.
 
 #### Tabular (tsv) metadata tables (Legacy)
-Four tsv files can be uploaded, one each for study, sample, experiment and run metadata. Example tsv files can be found [here](https://github.com/usegalaxy-eu/ena-upload-cli/tree/master/example_tables).
+Four tsv files can be uploaded, one each for study, sample, experiment and run metadata. Example tsv files can be found 
 Export the completed tables to tsv files and upload them using Galaxy upload tool.
 
-#### Select data and submit
+
+Complete the metadata template (spreadsheet and tsv tables) in your computer. **All fields of the template must be complete**. [Here](https://drive.google.com/file/d/1Z3LszV6IkgmcESsz2K7Mdv8pcnolUHT5/view?usp=sharing) you can find an example of part of the metadata associated with an actual submission to ENA (accession number PRJEB40711). Upload the metadata file to Galaxy, and select it on **'File based on templates here:'**.
```

Comment:
replace the google drive example link with the one in the github repo:
https://github.com/ELIXIR-Belgium/ENA-metadata-templates/blob/f16f9276e634cc0ab62f7482184ac004661961df/templates/ERC000033/example_metadata_template_ERC000033.xlsx